packages/admin-ui-form/src/resolvers/yup.ts

```diff
@@ -0,0 +1,3 @@
+import { yupResolver } from '@hookform/resolvers/yup'
```

Comment:
There are a lot of developers that use yup. This will make this available for them.